docs/content/grafana/README.md

```diff
@@ -1,16 +1,35 @@
 # Grafana
 
+<font size="4"> 
+
 [Grafana][1] is a multi-platform open source analytics and interactive visualization tool. It allows you to query, create dashboards, explore, and alert on metrics when connected to supported data sources.
 
-We use Grafana to visualize the metrics that we source from our [User Workload Monitoring][../uwm]. These monitoring dashboards are useful for visualizing the metrics (such as Prometheus metrics) collected from the applications to track and analyze the system's overall health.
+</font>
+
+<font size="3">
+
+We use Grafana to visualize the metrics that we source from our [User Workload Monitoring][10] (click to learn more). These monitoring dashboards are useful for visualizing the metrics (such as Prometheus metrics) collected from the applications to track and analyze the system's overall health.
```

Comment:
> (click to learn more)

Can we remove this, I think this part is self evident.

Reply:
Yes, leftover from original docs. Redundant now.